src/debugAdapter/goDebug.ts

```diff
@@ -36,6 +36,7 @@ import {
 	getInferredGopath,
 	parseEnvFile
 } from '../goPath';
+import { killTree } from '../util';
```

Comment:
We cannot have the debug adapter take a dependency on util which depends on the `vscode` module. This is because the debug adapter is run in a process different from the rest of the extension. The process in which the rest of the extension runs has access to `vscode`, but the process running the debug adapter doesnt. That's why previously, the kill code is copied here in this file instead of re-using from util

Reply:
We can however move the killTree out of the util into a separate file and then re-use it from there

Reply:
Ahh, got it.